2016-07-27/1.php

```diff
@@ -19,13 +19,7 @@
  * @return mixed[]
  */
 function arrayReverse($a) {
-    $reversed = [];
-
-    for ($i=count($a)-1; $i >= 0; $i--) {
-        $reversed[] = $a[$i];
-    }
-
-    return $reversed;
+    return array_reverse($a);
```

Comment:
Oh! Sorry, I wasn't aware there were any additional criteria on this project.
I'll remove this change so we can merge in the other improvement.